docs/js-core-1/week-2/lesson.md

```diff
@@ -176,43 +103,86 @@ You might see people use two other comparison operators. They are valid, but are
 
 If you see these, suggest people change them in pull requests.
 
-### Exercise (5 mins)
 
-1. Modify the values below to have the compare function return the desired value:
+### Exercise ( 15 mins )
+
+1. What do `typeof true` and `typeof false` evaluate to - write some code and log the output to the console to work out the answer.
+
+2. Check out the code below.
+
+Replace the variable `FILL_ME_IN` with actual values to prevent the `checkIsTrue` function from logging an error message.
 
 ```js
-const assert = require("assert");
+const mentorCount = FILL_ME_IN;
+const studentCount = FILL_ME_IN;
+checkIsTrue(mentorCount < studentCount);
 
-const mentorCount = // TODO
-const studentCount = // TODO
-assert(mentorCount < studentCount);
+const capacity = FILL_ME_IN;
+const people = FILL_ME_IN;
+checkIsTrue(capacity > people);
 
-const capacity = 25
-const people = // TODO
-assert(capacity > people);
+const organisation = FILL_ME_IN;
+checkIsTrue(typeof organisation === 'string')
+checkIsTrue(organisation.startsWith('Code'));
+checkIsTrue(organisation.includes(' your '));
+checkIsTrue(organisation.endsWith('Future!'));
 
-const name1 = // TODO
-const name2 = // TODO
-assert(name1 === name2);
+const firstName = FILL_ME_IN;
+const lastName = FILL_ME_IN;
+checkIsTrue(firstName[0] === lastName[0]);
```

Comment:
I think this is the first time trainees would encounter string indexing? Should we consider removing these lines and covering that explicitly? It can be confusing as trainees tend to think only of arrays when seeing indexing syntax.

Reply:
I personally don't think it is that tricky but am also happy to move it to an area where indexing is covered more explicitly